src/modules/error.js

```diff
@@ -323,6 +323,18 @@ export class NoOrgUnitResponseError extends VisualizationError {
     }
 }
 
+export class DataTypeError extends VisualizationError {
```

Comment:
"Data type" isn't really a thing, so I think we should call it `ValueTypeError`

Reply:
updated